ci-operator/config/openshift/insights-operator/openshift-insights-operator-master.yaml

```diff
@@ -72,29 +72,13 @@ tests:
     test:
     - as: test
       cli: latest
-      commands: make test-e2e
-      from: src
+      commands: pytest.sh -m "not ui"
+      from: insights-operator-tests
```

Comment:
@quarckster just out of curiosity. What is building and pushing the `quay.io/redhatqe/insights-operator-tests` image pls (I vaguely remember asking the question so sorry :) )? Gitlab CI?

Also the `pytest.sh -m "not ui"` - is there some UI option? :)

Reply:
The image is built here https://gitlab.cee.redhat.com/ccx/insights-operator-tests/-/blob/master/.gitlab-ci.yml. And, yes, these new tests can run UI tests against OCP web console. There three of them in the test suite.